### ISSUE TITLE
[Menus] Circular browsing

### ISSUE THREAD

**Issue by suazhero** [numworks/epsilon#1767]
#### Problem you'd like to fix
Menu browsing : Impossible to go directly to the last item of a list

#### Describe the solution you'd like
Allow the up arrow to bring the user down to the last item of the list (circular scrolling).

For example in "Settings", pressing the up arrow would bring the user directly to "About"
Same could apply for any other menu such as toolbox in calc app where pressing up arrow would bring the user directly to last item ("prediction interval").

#### Additional context
I acknoledge the fact that having such a circular browsing can be confusing compared to what currently exists, where the user is clearly able to detect what's the first and the last item of the list without having to remember the submenu names.
Maybe we could find a visual way to let the user understand it reached the bottom and is back to the first item? 
No clear opinion about that. Maybe a color that would blink every time the first item of the menu is selected?

**Comment by artaxxx** [comment]
Indeed, we decided not to do that because it was confusing. Especially in cases such as the table of values of a function. If you press `up` and go to the tab "Table" and then press `up` again, you'll send the selection at the bottom of the table of values whereas it's not even the same object.

The first step we decided to do is an acceleration of the selection when you keep `up` or `down` pressed, so you can navigate faster.

I'm not sure we'll add a circular navigation in the future.

**Comment by suazhero** [comment]
Didn't notice the "keep pressed" feature. That should make it then, will try on physical calc tonight. Thx

**Comment by woailafang** [comment]
@artaxxx Why not design a button for users to choose  Circular browsing or not?

**Comment by Stephan-P** [comment]
I'd like to bring this feature request to the attention again, because menu option lists are getting longer.

It will be very helpful to be able to scroll upwards through the list if you know that the option you need is at the end.
